starport-blog/04-http-create-post.md

```diff
@@ -0,0 +1,282 @@
+# Create posts with HTTP
+
+Let's create a front-end for our blog application. In this guide we will be writing a client-side application in JavaScript that can create a wallet (public/private key pair), fetch a list of posts from our server, create posts and send to our server.
+
+## `x/blog/client/rest/rest.go`
+
+```go
+import (
+  // Existing imports...
+  "github.com/example/blog/x/blog/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/client/utils"
```

Comment:
```suggestion
  "github.com/cosmos/cosmos-sdk/x/auth/client/utils"
```

Reply:
wy is this here actually? i don't see it being used.

Reply:
You're right, I'll remove it.